src/core/service.c

```diff
@@ -4455,13 +4468,14 @@ const UnitVTable service_vtable = {
                         [1] = "Stopping %s...",
                 },
                 .finished_start_job = {
-                        [JOB_DONE]       = "Started %s.",
+                        [JOB_DONE]       = NULL, /* handled specially in service_finished_job() */
```

Comment:
This comment is misleading, since the value that is used in the end may come either from service_finished_job() or the generic fallback. So I think it's better to fully remove this line including the comment.

Reply:
Removed.